junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTest.java

```diff
@@ -81,6 +80,8 @@
 	 *
 	 * @see java.text.MessageFormat
 	 */
+	//todo: consider either changing the default name or its interpretation:
+	// arguments = all arguments + arguments.description
 	String name() default "[{index}] {arguments}";
```

Comment:
How about `"[{index}] {arguments} {description}"` and trimming the name so an empty description would be ignored?

Reply:
I've replied in the issue, I currently believe it's better to expand `{arguments}` into two patterns, depending on whether an instance of `Arguments` has a description.